packages/btfs-http-client/examples/upload-file-via-browser/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "PeerID" : "16Uiu2HAm5KyUqXm7gZb5ECj4vDWVVjVuzNoj7cNM3Q7Eoz5Vsj2D",
+  "PrivKey": "CAISIAHU0I1KRRe0LvFEXXaIOT9aDa3RTSUc488JmHj4kCbW"
```

Comment:
Is it safe to have a private key in the repo?

Reply:
@rudya It will be up to the user to add their own key. These will be deleted before merging to master.

Reply:
Add `config.json` to .gitignore

Reply:
Okay, will do that!